### PR TITLE
Remove imports statements from project.json

### PR DESCRIPTION
**Pull request by rachelappel**
#1403

### CODE REVIEW

File: aspnet/security/app-secrets.rst [hunk 1]
```diff
@@ -37,12 +37,10 @@ Installing the Secret Manager tool
 
   "tools": {
     "Microsoft.AspNetCore.Razor.Tools": {
```
Comment (danroth27) [inline aspnet/security/app-secrets.rst]: ` "Microsoft.AspNetCore.Razor.Tools": "1.0.0-preview2-final" `

File: aspnet/security/app-secrets.rst [hunk 2]
```diff
@@ -37,12 +37,10 @@ Installing the Secret Manager tool
...
-      "version": "1.0.0-preview1-final",
-      "imports": "portable-net45+win8+dnxcore50"
+      "version": "1.0.0-preview1-final"
     },
     "Microsoft.Extensions.SecretManager.Tools": {
```
Comment (danroth27) [inline aspnet/security/app-secrets.rst]: ` "Microsoft.Extensions.SecretManager.Tools": "1.0.0-preview2-final" `

**Comment by Rick-Anderson** [comment]
@rachelappel  can you test this on an RTM machine. Double check the right string.

**Comment by Rick-Anderson** [comment]
@rachelappel  do you want me to set up an RTM VM for you? Can we get this fixed/merged today?

**Comment by rachelappel** [comment]
@Rick-Anderson ready to merge

**Pull request merged**